### ISSUE TITLE
Doesn't work with internet explorer 11

### ISSUE THREAD

**Issue by dhuber666** [rcaferati/react-awesome-slider#153]
Hi guys,

I think `react-awesome-slider` doesn't work with IE11. 
I just made a site for my customer and it won't work. Also the official examples and demos won't run on my machine IE 11.
Any quick fixes or workaround to get my slider working in IE11?

**The Problem**:

The first page of the slider open, but no arrows to navigate around. Autoplay also doesn't work. I can't see any errors in the console. 

His customers are mainly using IE11 (OMG).

Other then that awesome project!

Thx guys

**Comment by trolit** [comment]
Hi @dhuber666. If you visit css customizer <a href="https://caferati.me/demo/react-awesome-slider">here</a> provided by @rcaferati and scroll to the bottom of the page, you will see supported browsers.
**tl;dr;**
![supported browsers for ras](https://user-images.githubusercontent.com/25982919/99714715-39f18780-2aa6-11eb-8a7e-f4a8345584a1.png)
**tl;dr;2**
So if you want to keep up with this slider solution you will have to provide your own support for Internet Explorer 11.
